t/ui/10-tests_overview.t

```diff
@@ -102,6 +102,65 @@ sub schema_hook {
             ],
         });
     $jobs->find(99946)->update({result => OpenQA::Jobs::Constants::FAILED});
+
+    # add jobs for testing job template name
+    $jobs->create(
+        {
+            id         => 99990,
+            group_id   => 1002,
+            priority   => 30,
+            result     => OpenQA::Jobs::Constants::FAILED,
+            state      => OpenQA::Jobs::Constants::DONE,
+            TEST       => 'migration',
+            VERSION    => '13.1',
+            BUILD      => '0091',
+            ARCH       => 'x86_64',
+            MACHINE    => '64bit',
+            DISTRI     => 'opensuse',
+            FLAVOR     => 'DVD',
+            t_finished => time2str('%Y-%m-%d %H:%M:%S', time - 36000, 'UTC'),
+            t_started  => time2str('%Y-%m-%d %H:%M:%S', time - 72000, 'UTC'),
+            t_created  => time2str('%Y-%m-%d %H:%M:%S', time - 72000, 'UTC'),
+            settings   => [{key => 'JOB_TEMPLATE_NAME', value => 'migration_online'}],
+        });
+    $jobs->create(
+        {
+            id         => 99991,
+            group_id   => 1002,
+            priority   => 30,
+            result     => OpenQA::Jobs::Constants::PASSED,
+            state      => OpenQA::Jobs::Constants::DONE,
+            TEST       => 'migration',
+            VERSION    => '13.1',
+            BUILD      => '0091',
+            ARCH       => 'x86_64',
+            MACHINE    => '64bit',
+            DISTRI     => 'opensuse',
+            FLAVOR     => 'DVD',
+            t_finished => time2str('%Y-%m-%d %H:%M:%S', time - 36000, 'UTC'),
+            t_started  => time2str('%Y-%m-%d %H:%M:%S', time - 72000, 'UTC'),
+            t_created  => time2str('%Y-%m-%d %H:%M:%S', time - 72000, 'UTC'),
+            settings   => [{key => 'JOB_TEMPLATE_NAME', value => 'migration_offline'}],
+        });
+    $jobs->create(
+        {
+            id         => 99992,
+            group_id   => 1002,
+            priority   => 30,
+            result     => OpenQA::Jobs::Constants::INCOMPLETE,
+            state      => OpenQA::Jobs::Constants::DONE,
+            TEST       => 'migration',
+            VERSION    => '13.1',
+            BUILD      => '0091',
+            ARCH       => 'x86_64',
+            MACHINE    => '64bit',
+            DISTRI     => 'opensuse',
+            FLAVOR     => 'DVD',
+            t_finished => time2str('%Y-%m-%d %H:%M:%S', time - 36000, 'UTC'),
+            t_started  => time2str('%Y-%m-%d %H:%M:%S', time - 72000, 'UTC'),
+            t_created  => time2str('%Y-%m-%d %H:%M:%S', time - 72000, 'UTC'),
+            settings   => [{key => 'JOB_TEMPLATE_NAME', value => 'online_scc_base_all_full'}],
+        });
```

Comment:
this is quite a big copy-paste blob making it harder to maintain. Can you copy an existing job into the three ones? Or define a hash upfront which you reuse in all three occassions